nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-core/src/test/java/org/nuxeo/ecm/platform/audit/io/LogEntrySerializationTest.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * (C) Copyright 2017 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Funsho David
+ *
+ */
+
+package org.nuxeo.ecm.platform.audit.io;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.junit.Test;
+import org.nuxeo.ecm.core.api.impl.blob.StringBlob;
+import org.nuxeo.ecm.platform.audit.api.ExtendedInfo;
+import org.nuxeo.ecm.platform.audit.api.LogEntry;
+import org.nuxeo.ecm.platform.audit.impl.ExtendedInfoImpl;
+import org.nuxeo.ecm.platform.audit.impl.LogEntryImpl;
+
+import java.io.Serializable;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @since 9.3
+ */
+public class LogEntrySerializationTest {
+
+    protected static final String STRING_INFO = "stringInfo";
+
+    protected static final String LONG_INFO = "longInfo";
+
+    protected static final String DOUBLE_INFO = "doubleInfo";
+
+    protected static final String BOOL_INFO = "boolInfo";
+
+    protected static final String DATE_INFO = "dateInfo";
+
+    protected static final String BLOB_INFO = "blobInfo";
+
+    @Test
+    public void testLogEntrySerialization() throws Exception {
+
+        LogEntry entry = new LogEntryImpl();
+        entry.setId(1L);
+
+        Map<String, ExtendedInfo> extendedInfo = new HashMap<>();
+        extendedInfo.put(STRING_INFO, new ExtendedInfoImpl.StringInfo("this is an info"));
+        extendedInfo.put(LONG_INFO, new ExtendedInfoImpl.LongInfo(2L));
+        extendedInfo.put(DOUBLE_INFO, new ExtendedInfoImpl.DoubleInfo(2.0));
+        extendedInfo.put(BOOL_INFO, new ExtendedInfoImpl.BooleanInfo(true));
+
+        String inputString = "21-12-2012";
+        Date date = new SimpleDateFormat("dd-MM-yyyy").parse(inputString);
```

Comment:
Please use a test date that includes hours, minutes, seconds and also milliseconds.